package.json

```diff
@@ -78,25 +78,26 @@
     "@semantic-release/changelog": "^6.0.3",
     "@semantic-release/exec": "^6.0.3",
     "@semantic-release/git": "^10.0.1",
-    "@storybook/addon-a11y": "^6.5.16",
-    "@storybook/addon-actions": "^6.5.16",
-    "@storybook/addon-controls": "^6.5.16",
-    "@storybook/addon-docs": "^6.5.16",
-    "@storybook/addon-google-analytics": "^6.2.9",
-    "@storybook/addon-links": "^6.5.16",
-    "@storybook/addon-toolbars": "^6.5.16",
-    "@storybook/addon-viewport": "^6.5.16",
-    "@storybook/addons": "^6.5.16",
-    "@storybook/components": "^6.5.16",
-    "@storybook/react": "^6.5.16",
-    "@storybook/theming": "^6.5.16",
+    "@storybook/addon-a11y": "^7.6.4",
+    "@storybook/addon-actions": "^7.6.4",
+    "@storybook/addon-controls": "^7.6.4",
+    "@storybook/addon-docs": "^7.6.4",
+    "@storybook/addon-links": "^7.6.4",
+    "@storybook/addon-mdx-gfm": "^7.6.4",
```

Comment:
**issue**: because Storybook no longer natively supports GitHub-flavoured Markdown, we currently have a couple of broken tables in these docs:

- Documentation/Colors
- Grid
- Typography

![Colors docs with broken table](https://github.com/Sage/carbon/assets/18368713/bb1616ed-150a-43f5-95ba-d27230a198de)

To address this, what do you think about going with [Storybook's suggestion of using the `remark-gfm` plugin?](https://storybook.js.org/docs/writing-docs/mdx#lack-of-github-flavored-markdown-gfm):

```suggestion
    "remark-gfm": "^4.0.0"
```

Reply:
I'm going to have to come back to this one to work out what to do about it breaking. `remark-gfm` seems incompatible with our MDX files presumably because they're not MDX 2 yet. The `addon-mdx-gfm` file is a migration tool from Storybook while you wait to upgrade to MDX 2, so yes, ideally we wouldn't have it but for now we need it. But that doesn't answer how to fix the tables though...

Reply:
Okay I understand what the `addon-mdx-gfm` is now. It's a temporary package to do what remark-gfm can do but with some Storybook magic. They're trying to move away from Storybook having so much control of MDX files so it's still important to upgrade to MDX 2 and use the `remark-gfm` package. 

I have created this ticket for this work:

https://jira.sage.com/browse/FE-6390

For now though, this issue is resolved.